plugins/device-injector/device-injector.go

```diff
@@ -90,7 +93,7 @@ func (p *plugin) CreateContainer(_ context.Context, pod *api.PodSandbox, contain
 	}
 
 	if len(devices) == 0 {
-		log.Infof("%s: no devices annotated...", ctrName)
+		log.Debugf("%s: no devices annotated...", ctrName)
```

Comment:
Should we make the same change for `mounts`?

Reply:
Fixed in the refactored function.